packages/renderer/src/lib/volume/VolumeRow.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+import { router } from 'tinro';
+import StatusIcon from '../images/StatusIcon.svelte';
+import VolumeIcon from '../images/VolumeIcon.svelte';
+import VolumeActions from './VolumeActions.svelte';
+import type { VolumeInfoUI } from './VolumeInfoUI';
+
+export let object: any;
```

Comment:
I don't see why it should be with any type

Reply:
Fixed in latest commit.